feature/gnoi/system/tests/ping_test/ping_test.go

```diff
@@ -110,6 +111,11 @@ func TestGNOIPing(t *testing.T) {
 	if len(ipv6Addrs) == 0 {
 		t.Fatalf("Failed to get a valid IPv6 loopback address: %+v", ipv6Addrs)
 	}
+	if *deviations.ExplicitInterfaceInDefaultVRF {
+		if dut.Vendor() == ondatra.NOKIA {
```

Comment:
Can you help remove this change and file another PR for it? I want to have a separate discussion on it. Thanks.

Reply:
Sure, for now I reverted that commit.